### PR TITLE
fix(components/lookup): fix regression for lookup in modal

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#2656730](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2656730) - fixes issue introduced in #1494

### CODE REVIEW

File: libs/components/lookup/src/lib/modules/lookup/lookup.component.ts [hunk 1]
```diff
@@ -315,10 +316,6 @@ export class SkyLookupComponent
   #idService = inject(SkyIdService);
   #logSvc = inject(SkyLogService);
   #modalService = inject(SkyModalService);
-  #openNativePicker =
-    inject(SkyModalInstance, {
-      optional: true,
-    }) || undefined;
   #openSelectionModal =
```
Comment (Blackbaud-SteveBrush) [inline libs/components/lookup/src/lib/modules/lookup/lookup.component.ts]: Do we need to do the same thing for `#openSelectionModal`?

Comment (johnhwhite) [reply libs/components/lookup/src/lib/modules/lookup/lookup.component.ts]: Good catch. Updated.

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1656/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1656/storybooks/ag-grid-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/1656/storybooks/data-manager-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/1656/storybooks/lookup-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/1656/storybooks/phone-field-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1656/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1656/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1656/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1656/playground/)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/1657

**Pull request merged**